app-modules/portal/src/Filament/Pages/ManagePortalSettings.php

```diff
@@ -170,7 +173,19 @@ public function form(Form $form): Form
                             )
                             ->columnSpanFull(),
                     ])->columns(2),
-
+                Section::make('Header')
+                    ->schema([
+                        SpatieMediaLibraryFileUpload::make('logo')
+                            ->disk('s3')
```

Comment:
I know we've done this in a bunch of places previously, but to help future clean up please remove setting the disk here so we can rely on the `FILAMENT_FILESYSTEM_DISK` env key to handle the disk instead.

Reply:
@mxm1070 Correct me if I am wrong, so you are expecting changes as below?
->disk(env(FILAMENT_FILESYSTEM_DISK), 's3' )
And in env set FILAMENT_FILESYSTEM_DISK=s3

Reply:
@jayushi-canyon, no. You can just remove the `disk` method here. As Filament will just default to the default system disk. Which we currently have set to `s3`.

Reply:
@mxm1070 @Orrison Made the required changes.